openedx/core/djangoapps/oauth_dispatch/api.py

```diff
@@ -33,7 +33,15 @@ def create_dot_access_token(request, user, client, expires_in=None, scopes=None)
         request_validator=dot_settings.OAUTH2_VALIDATOR_CLASS(),
     )
     _populate_create_access_token_request(request, user, client, scopes)
-    return token_generator.create_token(request, refresh_token=True)
+
+    # save_token` has been deprecated, it was not called internally
+    # If you do, call `request_validator.save_token()`.
+    # for details https://github.com/oauthlib/oauthlib/blob/v3.2.0/oauthlib/oauth2/rfc6749/tokens.py#L303
+
```

Comment:
I'd remove this comment. It seems useful as a comment in the PR, but once this code change lands, the comment just is confusing.
```suggestion
```